boards/st/nucleo_n657x0_q/nucleo_n657x0_q.dts

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2024 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/n6/stm32n657X0.dtsi>
+#include <st/n6/stm32n657x0hxq-pinctrl.dtsi>
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+	model = "STMicroelectronics STM32N657X0-Q-NUCLEO board";
+	compatible = "st,stm32n657x0-q-nucleo";
+
+	chosen {
+		zephyr,console = &usart1;
+		zephyr,shell-uart = &usart1;
+		zephyr,sram = &axisram2;
+	};
+
+	leds: leds {
+		compatible = "gpio-leds";
+		green_led: led_1 {
```

Comment:
Still need an empty line above this node and the 2 nodes below.
Ditto at line 40.

Reply:
Done